ui/apps/platform/cypress/integration/policies/saveAsCustomResource.test.ts

```diff
@@ -0,0 +1,142 @@
+import * as api from '../../constants/apiEndpoints';
+import { selectors } from '../../constants/PoliciesPage';
+import { hasFeatureFlag } from '../../helpers/features';
+import withAuth from '../../helpers/basicAuth';
+import {
+    doPolicyPageAction,
+    doPolicyRowAction,
+    visitPolicies,
+    visitPolicy,
+} from '../../helpers/policies';
+import { interceptAndWatchRequests } from '../../helpers/request';
+
+const routeMatcherMapForPolicySaveAs = {
+    [api.policies.saveAs]: { method: 'post', url: api.policies.saveAs },
```

Comment:
What are pro and con to encapsulate the endpoint in the test file if used nowhere else?

Reply:
Small item: although `'post'` works, it seems unique compared to all-uppercase `method` values.

Sometimes, I use old-fashioned Find in Files like `method: 'POST'` so might overlook it.